src/enrichment_tables/file.rs

```diff
@@ -624,10 +623,10 @@ mod tests {
         };
 
         assert_eq!(
-            Ok(btreemap! {
-                "field1" => "zirp",
-                "field2" => "zurp",
-            }),
+            Ok(BTreeMap::from([
+                (String::from("field1"), Value::from("zirp")),
+                (String::from("field2"), Value::from("zurp")),
+            ])),
```

Comment:
I wonder if a new `btreemap` function might be useful to reduce the boilerplate with these. Something like:
```rust
fn btreemap<K: Into<String>, V: Into<Value>>(data: impl IntoIterator<(K, V)>) -> BTreeMap<String, Value>
```
Of course, with a different name than `btreemap` since we want to try out different results than a literal `BTreeMap`.

Reply:
Hmm, that is an interesting though. I had figured in places where we make `Value::Object` instances I'd swap out the `BTreeMap::from` for an `Object::from`, easy peasy but I'll give this suggestion a try when I do my next round of tree culling.